tests/reconfig/results/1.2.0/1.2.0.md

```diff
@@ -0,0 +1,87 @@
+# Reconfiguration testing Results
+
+<!-- TOC -->
+- [Reconfiguration testing Results](#reconfiguration-testing-results)
+  - [Summary](#summary)
+  - [Test environment](#test-environment)
+  - [Results Tables](#results-tables)
+    - [NGINX Reloads and Time to Ready](#nginx-reloads-and-time-to-ready)
+    - [Event Batch Processing](#event-batch-processing)
+  - [NumResources to Total Resources](#numresources-to-total-resources)
+  - [Observations](#observations)
+  - [Future Improvements](#future-improvements)
+<!-- TOC -->
+
+## Summary
+
+- Time to ready stayed consistent, if not slightly faster.
+- Reload time has slightly increased in some instances.
+- Number of batch events has reduced, subsequently increasing the size and time of each batch.
+
+## Test environment
+
+GKE cluster:
+
+- Node count: 3
+- Instance Type: e2-medium
+- k8s version: 1.27.8-gke.1067004
+- Zone: us-west2-a
+- Total vCPUs: 6
+- Total RAM: 12GB
+- Max pods per node: 110
+
+NGF deployment:
+
+- NGF version: edge - git commit 96a44240d317875406a8aef8fd1e424f2fb906eb
+- NGINX Version: 1.25.4
+
+## Results Tables
+
+> Note: After fixing the `cafe-tls-redirect` to point to the proper Gateway, tests that created 150 namespaces failed due to https://github.com/nginxinc/nginx-gateway-fabric/issues/1107. Therefore, those tests were re-run after reverting the `cafe-tls-redirect` issue to maintain consistency with the previous release tests. Going forward, results should look different once the above bug is fixed.
```

Comment:
#1107 has to do with match conditions being too long. Why would creating 150 namespaces trigger that error?

Reply:
I'll rephrase this. It's actually the increase in HTTPRoutes that we process.

Reply:
I still have the same question though... increasing the # of httproutes shouldn't affect the length of the matches

Reply:
Oh, maybe this is happening because the routes all have the same hostname and have a PathPrefix match?

Reply:
Yeah I think you're right.

Reply:
if the test generates matches, not simple prefix location, different reload time might be (speculatively) because of this https://github.com/nginxinc/nginx-gateway-fabric/commit/b2d078b88764d40f07ccadd83f74fcc090ffe769
although again, the difference is small

Reply:
We should change the path type to Exact so we aren't generating any matches for this test. The matches are tested by another NFR test.

Reply:
Doesn't change anything. We're getting an internal location created for each route with a redirect. It's a bunch of duplicates, so it doesn't appear that the path type matters.

Reply:
Although I don't see the matches being used anywhere...none of the regular location blocks refer to it

Reply:
Accidentally deleted my comment when trying to edit, but it looks like the path type doesn't matter, we just get a bunch of duplicate internal redirects.

```
location @rule0-route0 {
        return 302 "https://$host$request_uri";
    }

    location @rule0-route1 {
        return 302 "https://$host$request_uri";
    }

    location @rule0-route2 {
        return 302 "https://$host$request_uri";
    }

    location @rule0-route3 {
        return 302 "https://$host$request_uri";
    }
```

```
 location / {
        set $http_matches "[{\"redirectPath\":\"@rule0-route0\",\"any\":true},{\"redirectPath\":\"@rule0-route1\",\"any\":true},{\"redirectPath\":\"@rule0-route2\",\"any\":true}...
```

Reply:
Probably because we just create the same redirect route 150 times.

Reply:
hrm, ok. I think we need to look into this more post-release.